packages/web/docs/src/app/docs/[[...mdxPath]]/page.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { generateStaticParamsFor, importPage } from 'nextra/pages';
+import { useMDXComponents } from '../../../../mdx-components.js';
+import { ConfiguredGiscus } from '../../../components/configured-giscus';
+
+/**
+ * You might have an urge to try to refactor this to a separate file and reuse between product-updates and docs.
+ * I had the same urge. It's absurdly finicky. I warned you.
+ *
+ * BTW, even if we moved the product updates to page.mdx pattern, we still need this nesting to fix links in sidebar.
+ */
+export const generateStaticParams = async () => {
+  const pages = await generateStaticParamsFor('mdxPath')();
+  return pages
+    .map(page => (page.mdxPath[0] === 'docs' ? { mdxPath: page.mdxPath.slice(1) } : null))
+    .filter(Boolean);
+};
```

Comment:
use this new option who handles mdx `content` directory `basePath` https://github.com/shuding/nextra/blob/ffe0ad3c74b460cbb036f0592a5070a0927333a4/examples/docs/next.config.mjs#L8